great_expectations_provider/operators/great_expectations.py

```diff
@@ -78,24 +64,22 @@ class GreatExpectationsOperator(BaseOperator):
     ui_color = "#AFEEEE"
     ui_fgcolor = "#000000"
     template_fields = (
-        "checkpoint_name",
-        "batch_kwargs",
-        "assets_to_validate",
         "data_context_root_dir",
+        "data_context_config",
+        "checkpoint_name",
+        "checkpoint_config",
+        "checkpoint_kwargs",
     )
 
-    @apply_defaults
```

Comment:
Removing this should really mean that the lower bound of the Airflow version to use this provider is 2.1.0 rather than 2.0.